server/src/tasks/eth_funder.js

```diff
@@ -32,13 +36,13 @@ const ethFunder = withAccount(async (account, { accountAddress }, job) => {
   })
   if (receipt) {
     job.attrs.data.receipt = true
-    account.nonces['foreign']++
+    account.nonces[bridgeType]++
     await setFunded(ethFunding._id)
     await account.save()
     job.save()
   }
-}, (args) => {
-  return lockAccount({ role: 'eth' })
+}, ({ networkName }) => {
+  return lockAccount({ role: networkName === 'fuse' ? '*' : 'eth' })
```

Comment:
looking at code it's kinda weird to lock different account roles. But that's the fastest way to use the default account